test/e2e/k8s_util.go

```diff
@@ -941,6 +941,16 @@ func (data *TestData) DeleteANP(ns, name string) error {
 	return nil
 }
 
+// DeleteClusterRole is a convenience function for deleting DeleteClusterRole by name.
+func (data *TestData) DeleteClusterRole(name string) error {
```

Comment:
Shall we move such operations on ClusterRole/RBAC to the yaml ?

Reply:
You mean doing this operation by `kubectl delte -f ....yaml`. What't the benefit of that?

Reply:
We already configured the RBAC in this file https://github.com/antrea-io/antrea/blob/main/build/yamls/externalnode/support-bundle-collection-rbac.yml , we can use it in the CI test, and what we shall do is to mofify the test Namespace if it is different from the sample yaml. Such configuration items should not be included in the test code, right?

Reply:
updated. Thanks